### DISCUSSION TITLE
[PSD1] Exercise 1.14

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/psd2223#111] [Esercizi (A-L)]
**Exercise 1.14** Convert the following unsigned binary numbers to decimal. Show your work.

(a) $1110_2$
(b) $100100_2$
(c) $11010111_2$
(d) $011101010100100_2$

**Comment by niccolo-fato** [comment]
![1 14](https://user-images.githubusercontent.com/73995334/210086401-3c7fd22a-414b-418a-9754-357625bcb35e.jpg)
